howtos/pubsub/how_to_use_local_transactions.py

```diff
@@ -0,0 +1,153 @@
+                    publisher.publish(message, topic)
+                # Publish all messages to a topic, or none at all:
+                transactional_service.commit()
+                # Do not count messages until the commit succeeds.
+                messages_delivered += count
+            except TransactionRollbackError:
+                # Commit failed.
+                pass
+            except UnknownTransactionStateError:
+                # Commit outcome unknown. This should be very rare.
+                pass
+
+        transactional_service.disconnect()
+        return messages_delivered
+
+
+    @staticmethod
+    def receive_from_queues(service, queues, count):
+        """ Receive a message from each queue in a transaction, <count> times.
+            Each count is a separate transaction: A message is taken from every queue or none. """
+
+        messages = []
+        receivers = []
+        transactional_service = service.create_transactional_service_builder().build().connect()
+        # Start a receiver for every queue
+        for queue in queues:
+            receiver = transactional_service.create_transactional_message_receiver_builder().build(queue).start()
+            receivers.append(receiver)
+
+        for _ in range(count):
+            # Grab a message from each receiver/queue
+            messages_in_this_transaction = []
+            for receiver in receivers:
+                message = receiver.receive_message()
+                if message is None:
+                    if messages_in_this_transaction:
+                        # One of the queues ran empty, rolling back the partial transaction:
+                        transactional_service.rollback()
+                    break
+                messages_in_this_transaction.append(message)
+            else: # no break
+                try:
+                    # Consume a message from all queues, or from none:
+                    transactional_service.commit()
+                    messages.extend(messages_in_this_transaction)
+                except TransactionRollbackError:
+                    # Commit failed.
+                    pass
+                except UnknownTransactionStateError:
+                    # Commit outcome unknown. This should be very rare.
+                    pass
+        transactional_service.disconnect()
+        return messages
+
+
+    @staticmethod
+    def run():
+        """ Execute the demonstrator methods. """
+        try:
+            number_of_queues = 5
+            messages_per_queue = 3
+            messaging_service = MessagingService.builder().from_properties(boot.broker_properties()).build()
+            messaging_service.connect()
+            print(f'Message service is connected? {messaging_service.is_connected}')
+            topics = []
+            queues = []
+            for i in range(number_of_queues):
+                topic_name = constants.TOPIC_ENDPOINT_DEFAULT + str(i)
```

Comment:
Can we use `constanst.TOPIC_ENDPOINT` instead to advocate for best practices with topics using hierarchical levels. thanks!

Reply:
Done.
It's value is "purchase/tickets/local_transaction_sample", but it's centrally configurable, so I'm not worried.